usmqe_tests/conftest.py

```diff
@@ -442,7 +442,7 @@ def unmanaged_cluster(valid_session_credentials):
     cluster is not *managed*.
     """
     cluster_reuse = get_cluster_reuse(valid_session_credentials)
-    if cluster_reuse["is_managed"] != "no":
+    if cluster_reuse["is_managed"] == "yes":
```

Comment:
@fbalak did I understand it well, that this value comes from Tendr API (...from etcd) and if the cluster is managed, it will always contain `yes`, but if it is not managed it might be empty or contain `no`?

Reply:
Yes, it seems that there can be empty value at some times. I am not sure if it happens only after installation or it can occur also right after unmanage.